gcp/apis/compute/v1alpha1/network_types.go

```diff
@@ -0,0 +1,331 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"encoding/json"
+	"net/http"
+
+	googlecompute "google.golang.org/api/compute/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/crossplaneio/crossplane/apis/core/v1alpha1"
```

Comment:
This will need to be updated to `runtimev1alpha1 "github.com/crossplaneio/crossplane-runtime/apis/core/v1alpha1"` and all `v1alpha1.*` will be converted to `runtimev1alpha1.*`. You should see errors on here whenever you rebase.

Reply:
Thanks for the heads-up! I think I'll save that fix for the end where I'd fix the conflicts